app/models/job.rb

```diff
@@ -52,6 +52,54 @@ def pct_format(number)
       # Rails.logger.info("formatting #{number} to #{(number * 100).to_i}")
       (number * 100).to_i
     end
+
+    def app_inspector_data
+      apps = {}
+      Job.all.each do |job|
+        next unless job.tres
+
+        cpus = job.tres.match(/cpu=(\d*)/).captures[0].to_i
```

Comment:
I think `procs` could be reliable way to get all the cores.

```suggestion
        cpus = job.procs
```

Reply:
Sorry - Im getting confused I thought we were interacting with `OodCore::Job::Info` and now realize their the `Job` Model that we made in the database.

I'll open a ticket to directly put `cores` into the DB.